### PR TITLE
Add Flexport as Sorbet user

### PR DESCRIPTION
**Pull request by dylanirlbeck**
Adding [Flexport](https://flexport.com) as a Sorbet user. 

### Motivation
[We've been using](https://flexport.engineering/adding-sorbet-to-a-rails-monolith-ef72d6a18449) Sorbet for a few years now and it has worked rather well! We'd love to make our usage of this awesome tool known to the public.

### Test plan
Screen Shot 2021-12-25 at 11 33 09 PM

See included automated tests.

### Questions

I noticed that there are a few "early adopters" of this tool that appear early in the `users` array. Would we qualify here? I'm a fairly new engineer here so I don't have much context. Thanks!

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ const users = [
     image: 'img/factorial-logo.png',
     infoLink: 'https://factorialhr.com/',
   },
+  {
+    caption: 'Flexport',
+    image: 'img/flexport-logo.jpg',
+    infoLink: 'https://www.workforce.com/'
```
Comment (dylanirlbeck) [inline website/siteConfig.js:42]:
```suggestion
    infoLink: 'https://flexport.com/'
```

**Review by froydnj — APPROVED** [review]
Cool, thank you!

**Comment by froydnj** [comment]
> I noticed that there are a few "early adopters" of this tool that appear early in the users array. Would we qualify here? I'm a fairly new engineer here so I don't have much context. Thanks!

👋  I'm a new(ish) engineer here too, so I don't know exactly what the "early adopter" criteria is.  I'll merge this as-is and ask around, and we can fix up later if need be.

File: website/siteConfig.js [hunk 2]
```diff
@@ -36,6 +36,11 @@ const users = [
     image: 'img/factorial-logo.png',
     infoLink: 'https://factorialhr.com/',
   },
+  {
+    caption: 'Flexport',
+    image: 'img/flexport-logo.jpg',
+    infoLink: 'https://flexport.com/'
```
Comment (froydnj) [inline website/siteConfig.js:42]:
```suggestion
    infoLink: 'https://flexport.com/',
```

**Pull request merged**